FileWritingObjects/FileWritingObjects/SetWriter.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+
+namespace FileWritingObjects
```

Comment:
This should be a nested namespace.
```suggestion
namespace FileWritingObjects.Nested
```